config/initializers/paperclip.rb

```diff
@@ -6,4 +6,7 @@
     secret_access_key: ENV['AWS_SECRET_ACCESS_KEY'],
     s3_region: ENV['S3_REGION']
   }
+  Paperclip::HttpUrlProxyAdapter.register
+elsif Rails.env.development?
+  Paperclip::HttpUrlProxyAdapter.register
```

Comment:
What is this for?

Reply:
This is for Paperclip to get the image from an URL instead of the form. That's what I'm using in the importer.

Reply:
Can you add a comment to make it clear? :)

Reply:
Done